File: sample-codes-for-test/a-fizzbuzz/fizzbuzz.php
<?php

$N = (int)(trim(fgets(STDIN)));

for ($i = 0; $i < $N; ++$i) {
	$t = (int)(trim(fgets(STDIN)));

	if ($t % 15 == 0) {
		print "FizzBuzz\n";
	} else if ($t % 5 == 0) {
		print "Buzz\n";
	} else if ($t % 3 == 0) {
		print "Fizz\n";
	} else {
		print $t . "\n";
	}
}

File: sample-codes-for-test/a-fizzbuzz/fizzbuzz.rb
N = (STDIN.gets.chomp).to_i

for i in 1..N do
	t = (STDIN.gets.chomp).to_i

	if t % 15 == 0 then
		puts "FizzBuzz"
	elsif t % 5 == 0 then
		puts "Buzz"
	elsif t % 3 == 0 then
		puts "Fizz"
	else
		puts t
	end
end
